public/app/features/explore/QueryEditor.tsx

```diff
@@ -73,7 +73,7 @@ export default class QueryEditor extends PureComponent<QueryEditorProps, any> {
     const hasNewError = prevProps.error !== this.props.error;
 
     if (this.component) {
-      if (hasToggledEditorMode) {
+      if (hasToggledEditorMode && this.angularScope && this.angularScope.toggleEditorMode) {
```

Comment:
think we should use the same logic as in the dashboard query editor row component: https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/panel_editor/QueryEditorRow.tsx#L86

Reply:
Okay. I can change. I was following this line: https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/panel_editor/QueryEditorRow.tsx#L164

Reply:
ah, I see there is a runtime check as well